ok_tools/urls.py

```diff
@@ -14,12 +14,14 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
+from django.contrib.auth.decorators import login_required
```

Comment:
As the dotted name implies `login_required` is intended to be used as a decorator. This is typically done in the `views.py` as then all information about the view are close together. Applying `login_required` is a security feature, which should be clearly mentioned next to the view definition. 

In my opinion, the `urls.py` should only include the most basic information to wire the views up, but not the actual logic of the views.